Fluid/DefaultFluidParser.cs

```diff
@@ -1018,7 +1033,8 @@ public static FilterArgument BuildFilterArgument(ParseTreeNode node)
         {
             string identifier = null;
             Expression term = null;
-            if (node.ChildNodes[0].Term.Name == "identifier")
+
+            if (String.Equals(node.ChildNodes[0].Term.Name, "identifier", StringComparison.Ordinal))
```

Comment:
I think I once benchmarked equality and raw `==` seemed to be a solid choice, YMMV